cdc/pkg/cmd/cli/cli_changefeed_create.go

```diff
@@ -72,6 +82,10 @@ func (o *changefeedCommonOptions) addFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&o.sortEngine, "sort-engine", model.SortUnified, "sort engine used for data sort")
 	cmd.PersistentFlags().StringVar(&o.sortDir, "sort-dir", "", "directory used for data sort")
 	_ = cmd.PersistentFlags().MarkHidden("sort-dir")
+	cmd.PersistentFlags().StringVar(&o.format, flagKeyFormat, "hex", "The format of start and end key. Available options: \"raw\", \"escaped\", \"hex\".")
+	cmd.PersistentFlags().StringVar(&o.startKey, flagStartKey, "", "The start key of the changefeed, key is inclusive.")
+	cmd.PersistentFlags().StringVar(&o.endKey, flagEndKey, "", "The end key of the changefeed, key is exclusive.")
+
```

Comment:
If user don't input 'start-key' and 'end-key', can we get a defaut 'start-key' and 'end-key'?

Reply:
Default is empty and we encode it to [r000, r001) here: https://github.com/haojinming/migration/blob/6f40097eedf69f64e50179d58776ff3810f28976/cdc/pkg/util/key.go#L27